package.json

```diff
@@ -4,14 +4,15 @@
   "keywords": [
     "assert"
   ],
-  "version": "1.3.0",
+  "version": "1.4.0",
   "homepage": "https://github.com/defunctzombie/commonjs-assert",
   "repository": {
     "type": "git",
     "url": "git://github.com/defunctzombie/commonjs-assert.git"
   },
   "main": "./assert.js",
   "dependencies": {
+    "buffer": "^4.6.0",
```

Comment:
Please don't do dependencies like this. Dependencies should be specified exactly.

Also, this bloats this module a lot.